### PR TITLE
Add Buildifier Lint

### PR DESCRIPTION
**Pull request by loganmcbride148**
Add Buildifier Lint capabilites and a test to verify. Change some files to exclude 'Buildifier' prefix.

### CODE REVIEW

File: examples/cpp_hello_world/stage_3/main/BUILD [hunk 1]
```diff
@@ -13,4 +13,5 @@ cc_binary(
         ":hello_greet",
         "//lib:hello_time",
     ],
+    bad
```
Comment (josiahsrc) [inline examples/cpp_hello_world/stage_3/main/BUILD]: We shouldn't commit breaking changes to the examples

**Review by josiahsrc — APPROVED** [review]
One comment about the example folder. Other than that, solid work!

File: examples/cpp_hello_world/stage_3/main/BUILD (hunk 1 above)
Comment (loganmcbride148) [reply examples/cpp_hello_world/stage_3/main/BUILD]: Good catch. I just fixed it. That was just my leftover from trying different things.

**Review by jasonxris — APPROVED** [review]
👌

**Review by NovaMachina — APPROVED** [review]

**Review by jarenm — APPROVED** [review]
🙌

**Pull request merged**